jekyll/_cci2/set-up-multiple-configuration-files-for-a-project.adoc

```diff
@@ -29,23 +29,25 @@ In the GitHub repository for your project add a new CircleCI configuration file
 
 As your repository is set up as a project in CircleCI, a `config.yml` file will exist already. If you do not see `.circleci/config.yml`, check which branch you are on. You may have set this up on a non-default branch during the set up process.
 
-Each configuration file will trigger a separate pipeline so it is important that each configuration file is complete with all necessary elements for the pipeline, for example, `version`, `jobs`, `workflows`, `commands`, `executors`, `orbs` etc.
+Each configuration file can be set up with a pipeline and trigger so it is important that each configuration file is complete with all necessary elements for the pipeline, for example, `version`, `jobs`, `workflows`, `commands`, `executors`, `orbs` etc.
```

Comment:
The old version of this is so misleading and poorly worded that I think it's influencing the new version. I might try something different

Reply:
Each of your configuration files represents the full set of commands necessary for a pipeline to be executed, so it should be complete with all necessary elements - for example, `version`, `jobs`, `workflows`, `commands`, `executors`, `orbs` etc.